apps/src/calc/calc.js

```diff
@@ -80,6 +81,12 @@ function getTokenList(one, two) {
   if (two instanceof Equation) {
     two = two.expression;
   }
+  if (jsnums.isSchemeNumber(one)) {
```

Comment:
This function is becoming slightly challenging to reason about what the typical execution flow would be & what might be dependent on what. I.e., these two clauses probably won't convert `one` into a `string` (so the later check would not apply), but in either the inputs or after the expression unwrapping, they might then become strings & tokenized.

One potential change could be to have a helper e.g. `getExpressionOrStringFrom(expressionOrEquationOrSchemeNumber)` which does what lines 81-89 do, then have `one = getExpressionFrom(one); two = getExpressionFrom(two);`. The jsdoc @param annotation on that would then hint about the types that can be massaged.

Or maybe just adding a jsdoc `@param` hint about the potential types of `one` and `two` for this method could help.